en/identity-server/next/docs/guides/your-is/manage-console-access.md

```diff
@@ -1 +1,22 @@
+{% set wso2_is_console_access_note = "
+## Accessing the WSO2 Identity Server Console
+
+To access the WSO2 Identity Server Console, you need to have the required permissions and the URL of the Console. This can be achieved by navigating to the console through a web browser using the following URL format:
```

Comment:
```suggestion
To access the {{product_name}} Console, you need to have the required permissions and the URL of the Console. This can be achieved by navigating to the console through a web browser using the following URL format:
```

Reply:
Placeholders are not resolvable in the places we set the values to variables. Since this is IS specific page having the product name will not be a problem.